test/harmony-regression/HarmonyRegression.ipynb

```diff
@@ -60,7 +61,9 @@
    "source": [
     "coverages_root = '{root}/{collection}/ogc-api-coverages/1.0.0/collections/{variable}/coverage/rangeset'\n",
     "\n",
-    "if harmony_host_url == 'https://harmony.uat.earthdata.nasa.gov' or harmony_host_url == 'https://harmony.sit.earthdata.nasa.gov':\n",
+    "is_uat_or_sit = harmony_host_url == 'https://harmony.uat.earthdata.nasa.gov' or harmony_host_url == 'https://harmony.sit.earthdata.nasa.gov' or re.search(\"http:\\/\\/internal-harmony-.*us-west-2\\.elb.amazonaws\\.com\", harmony_host_url)\n",
```

Comment:
This will also return true for sandbox. I think what we're going for is whether it's prod or not, so maybe change the name to something other than `is_uat_or_sit`

Reply:
Changed to is_not_prod